server/src/domain/metadata/metadata.service.ts

```diff
@@ -311,7 +311,7 @@ export class MetadataService {
         assetId: asset.id,
         bitsPerSample: this.getBitsPerSample(tags),
         colorspace: tags.ColorSpace ?? null,
-        dateTimeOriginal: exifDate(firstDateTime(tags as Tags)) ?? asset.fileCreatedAt,
+        dateTimeOriginal: exifDate(tags.CreationDate ?? firstDateTime(tags as Tags)) ?? asset.fileCreatedAt,
```

Comment:
Can you instead do something like `firstDateTime(tags as Tags, <updated list here>)` to include your proposed list change instead?